zilencer/management/commands/populate_db.py

```diff
@@ -253,16 +253,15 @@ def handle(self, **options: Any) -> None:
 
         # Create a test realm emoji.
         IMAGE_FILE_PATH = os.path.join(settings.STATIC_ROOT, 'images', 'test-images', 'checkbox.png')
-        UPLOADED_EMOJI_FILE_NAME = 'green_tick.png'
         with open(IMAGE_FILE_PATH, 'rb') as fp:
-            upload_backend.upload_emoji_image(fp, UPLOADED_EMOJI_FILE_NAME, iago)
-            RealmEmoji.objects.create(
-                name='green_tick',
-                author=iago,
-                realm=zulip_realm,
-                deactivated=False,
-                file_name=UPLOADED_EMOJI_FILE_NAME,
-            )
+            realm_emoji = RealmEmoji.objects.create(name='green_tick',
+                                                    author=iago,
+                                                    realm=zulip_realm,
+                                                    deactivated=False)
+            file_name = str(realm_emoji.id) + '.png'
+            upload_backend.upload_emoji_image(fp, file_name, iago)
+            realm_emoji.file_name = file_name
+            realm_emoji.save(update_fields=['file_name'])
```

Comment:
This might be able to just use `check_add_realm_emoji`, once we do the changes I suggest above.